cpp/examples/sbml2memilio.cpp

```diff
@@ -0,0 +1,53 @@
+#include "sbml2memilio.h"
+
+int main(int argc, char* argv[])
+{
+    if (argc != 2) {
+        std::cout << "Please provide a SBML file at startup!" << std::endl << std::endl;
+        return 1;
+    }
+    const char* filename = argv[1];
+    SBMLDocument* document;
```

Comment:
do we need to use a raw pointer here?

Reply:
Looking at SBMLReader.cpp, the pointer returned by readSBML is owning and should therefore be accepted as a uniqe_ptr:
```
std::unique_ptr<SBMLDocument> document(reader.readSBML(filename));
```

The Model*, however, is non-owning, so we should accept it as a (const, if possible) Model&, i.e.
```
Model& model = *document->getModel();
```
The function arguments should be changed accordingly.

Reply:
Also, the library uses class, virtual, and std::string, it is very much C++ (or has a C++ interface). But anyway, we should not use raw pointers, if avoidable.